pki/deploy.go

```diff
@@ -62,8 +62,8 @@ func DeployCertificatesOnWorkers(workerHosts []*hosts.Host, crtMap map[string]Ce
 }
 
 func doRunDeployer(host *hosts.Host, containerEnv []string, certDownloaderImage string) error {
-	logrus.Debugf("[certificates] Pulling Certificate downloader Image on host [%s]", host.Address)
-	if err := docker.PullImage(host.DClient, host.Address, certDownloaderImage); err != nil {
+	logrus.Debugf("[certificates] Checking image [%s] on host [%s]", certDownloaderImage, host.Address)
```

Comment:
same as before, can you remove the `Checking image` logs from all calls and add it instead to the UseLocalOrPull